k8s/eks/testground_install.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+set -e # Let's make sure that exit code 1 drops the script
+start=$(date +"%Y-%m-%d-%T")
+source ./bash/functions.sh
+prep_log_dir
+cat << "EOF"
+ _____         _   _____                           _ 
+|_   _|       | | |  __ \                         | |
+  | | ___  ___| |_| |  \/_ __ ___  _   _ _ __   __| |
+  | |/ _ \/ __| __| | __| '__/ _ \| | | | '_ \ / _` |
+  | |  __/\__ \ |_| |_\ \ | | (_) | |_| | | | | (_| |
+  \_/\___||___/\__|\____/_|  \___/ \__,_|_| |_|\__,_|
+                  EKS setup script 
+
+
+EOF
+
+source .env # Setting .env vars
+echo "Log path is ./log/$start-log/cluster.log "
+# Let's do a sanity check
+if [ -n "$cluster_setup_init" ]; then
```

Comment:
The way we mutate the `.env` file is fragile and confusing:

One example: this feature breaks when you change the `CLUSTER_NAME` (it'll find a cluster that actually doesn't exist or vice-versa).

- Can we use a more explicit name? (`CLUSTER_WAS_INITIALIZED` for example),
- Is there a way to detect the cluster without relying on a mutation of the `.env` file?

If we can't make this feature reliable, I would recommend we drop it and use a simple `start` / `delete`

Reply:
Could you please explain the mutation part? And also the explicit name as I don't really understand what improvement would that introduce, and I cannot answer this question until I undestand what you meant by this. Thanks.

Reply:
Aha ok, I think I understand now, thanks for the clarification.
As discussed somewhere in these comments, we will look into removing the part that writes to the `.env` file.
And also will work on setting up a more reliable mechanism of cluster detection, rather than the `cluster_setup_init`.
Please remember that this is just the first version and still a work in progress.